packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -163,14 +175,15 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
     );
 
     const overflowStyle: React.CSSProperties = overflowYHidden ? { overflowY: 'hidden' } : {};
+    // React.CSSProperties does not understand IRawStyle, so the inline animations will need to be cast as any for now.
     const content = (
       <div
         ref={ this._resolveRef('_hostElement') }
         className={ this._classNames.container }
       >
         <div
-          className={ this._classNames.root }
-          style={ positions ? positions.elementPosition : OFF_SCREEN_STYLE }
+          className={ css(this._classNames.root, positions && positions.targetEdge && ANIMATIONS[positions.targetEdge!]) }
+          style={ positions ? { ...positions.elementPosition } : OFF_SCREEN_STYLE }
```

Comment:
`positions.elementPosition` and `{ ...positions.elementPosition }` are identical no? Is there some immutability thing you're fighting?